src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -100,35 +76,9 @@ export const TradeConfirm = ({ history }: RouterProps) => {
       }
     } catch (err) {
       console.error(`TradeConfirm:onSubmit - ${err}`)
-      let errorMessage
-      switch ((err as ZrxError).message) {
-        case 'ZrxSwapper:ZrxExecuteTrade Cannot execute a failed quote': {
-          errorMessage = TRADE_ERRORS.FAILED_QUOTE_EXECUTED
-          break
-        }
-        case 'ZrxSwapper:ZrxExecuteTrade sellAssetAccountId is required': {
-          errorMessage = TRADE_ERRORS.SELL_ASSET_REQUIRED
-          break
-        }
-        case 'ZrxSwapper:ZrxExecuteTrade sellAmount is required': {
-          errorMessage = TRADE_ERRORS.SELL_AMOUNT_REQUIRED
-          break
-        }
-        case 'ZrxSwapper:ZrxExecuteTrade depositAddress is required': {
-          errorMessage = TRADE_ERRORS.DEPOSIT_ADDRESS_REQUIRED
-          break
-        }
-        case 'ZrxSwapper:ZrxExecuteTrade sellAssetNetwork and sellAssetSymbol are required': {
-          errorMessage = TRADE_ERRORS.SELL_ASSET_NETWORK_AND_SYMBOL_REQUIRED
-          break
-        }
-        default: {
-          errorMessage = getParametrizedErrorMessageOrDefault((err as ZrxError).message)
-        }
-      }
       toast({
         title: translate('trade.errors.title'),
-        description: translate(errorMessage),
+        description: translate(TRADE_ERRORS.DEX_TRADE_FAILED),
```

Comment:
does this make things significantly worse from a UX perspective?

Reply:
no. 

These would have been checked by validation in web before it even gets to swapper.

Swapper doesnt even check those values to throw an error anymore because they are required. Really we should just need the general dex_trade_failed message because if something fails it wont be because the user did anything wrong like not entering an amount